third-party/cmake/BuildLuarocks.cmake

```diff
@@ -167,15 +167,15 @@ if(USE_BUNDLED_BUSTED)
   # DEPENDS on the previous module, because Luarocks breaks if parallel.
   add_custom_command(OUTPUT ${BUSTED_EXE}
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build https://raw.githubusercontent.com/Olivine-Labs/busted/v2.0.rc12-1/busted-2.0.rc12-1.rockspec ${LUAROCKS_BUILDARGS}
+    ARGS build busted ${LUAROCKS_BUILDARGS}
```

Comment:
They are slow to release, so we can't do this for busted. We need bleeding-edge busted for certain bugfixes.

Reply:
This is installing the same version of busted as before the change -- the last release.